sdk/cosmos/azure_data_cosmos/src/clients/cosmos_client.rs

```diff
@@ -96,6 +96,37 @@ impl CosmosClient {
         })
     }
 
+    /// Creates a new CosmosClient, using a connection string.
+    ///
+    /// # Arguments
+    ///
+    /// * `connection_string` - the connection string to use for the client, e.g. AccountEndpoint=https://accountname.documents.azure.com:443/‌​;AccountKey=accountk‌​ey
```

Comment:
We should wrap the connection string in backticks so it renders in monospace font.

```suggestion
    /// * `connection_string` - the connection string to use for the client, e.g. `AccountEndpoint=https://accountname.documents.azure.com:443/‌​;AccountKey=accountk‌​ey`
```